setup.cfg

```diff
@@ -0,0 +1,29 @@
+[metadata]
+name = zcash-resp-time-MANAN-MONGA
+version = 0.0.1
+author = Manan Monga
+author_email = author@example.com
+description = A small example package
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/pypa/sampleproject
+project_urls = 
+	Bug Tracker = https://github.com/pypa/sampleproject/issues
+classifiers = 
+	Programming Language :: Python :: 3
+	License :: OSI Approved :: MIT License
+	Operating System :: OS Independent
+
+[options]
+package_dir = 
+	= src
```

Comment:
Is this correct?

Reply:
I think so, not sure what it should be pointing to